src/core/builder/builder-base.ts

```diff
@@ -25,14 +26,20 @@ export abstract class BaseBuilder implements IBuilder {
       return;
     }
 
-    var files = readdirSync(startPath);
-    for (var i = 0; i < files.length; i++) {
-      var filename = join(startPath, files[i]);
+    const files = readdirSync(startPath);
+    for (let i = 0; i < files.length; i++) {
+      const filename = join(startPath, files[i]);
       //skip the actual thirdweb contract itself
       if (basename(filename, ".json") === "ThirdwebContract") {
         continue;
       }
-      var stat = statSync(filename);
+      const stat = statSync(filename);
+
+      // brownie has a "depdendencies" directory *inside* the build directory, if we detect that we should skip it
+      if (stat.isDirectory() && basename(filename) === "dependencies") {
```

Comment:
hopefully this base class handling doesnt get crazier than this - otherwise might have to pull it up

Reply:
yeh